libs/backend-api7/src/transformer.ts

```diff
@@ -188,9 +192,9 @@ export class FromADC {
     serviceId: string,
   ): typing.StreamRoute {
     return ADCSDK.utils.recursiveOmitUndefined<typing.StreamRoute>({
-      stream_route_id: route.id,
+      stream_route_id: route.id || '',
       name: route.name,
-      desc: route.description,
+      desc: route.description || '',
```

Comment:
Ensure non-critical fields remain undefined, otherwise recursiveOmitUndefined will lose its purpose and distort the original input values.